examples/batched-bench/batched-bench.cpp

```diff
@@ -41,7 +41,6 @@ int main(int argc, char ** argv) {
     int n_kv_max     = 2048;
     int is_pp_shared = 0;
     int n_gpu_layers = 0;
-    int mmq          = 0;
```

Comment:
Also remove from the `usage` prints above

Reply:
Done. @ggerganov By the way what is the HF repo and file to run the `llama-bench`  please ?

```shell
./llama-bench -o sql | sqlite3 llama-bench.sqlite
main: error: failed to load model 'models/7B/ggml-model-q4_0.gguf'
```

Reply:
You can use any model with `llama-bench`, the default is supposed to be llama-7B Q4_0, but it is not important to use that model.